python/pylibraft/pylibraft/neighbors/ivf_pq/ivf_pq.pyx

```diff
@@ -722,3 +723,90 @@ def search(SearchParams search_params,
         raise ValueError("query dtype %s not supported" % queries_dt)
 
     return (distances, neighbors)
+
+
+@auto_sync_handle
+def save(filename, Index index, handle=None):
```

Comment:
We should probably note very clearly in the pydocs that this feature is currently experimental and subject to change (so it's possible they might not be able to load what they saved should changes occur).